src/Package/Impl/Utilities/ProjectUtilities.cs

```diff
@@ -42,5 +52,95 @@ public static class ProjectUtilities {
 
             return result;
         }
+
+        public static EnvDTE.Project GetActiveProject() {
```

Comment:
Since it uses GetGlobalService, it is easily replaceable in tests by mock. Doesn't have to be passed, see
```
        private TestVsAppShell() {
            CompositionService = VsTestCompositionCatalog.Current.CompositionService;
            ExportProvider = VsTestCompositionCatalog.Current.ExportProvider;
            MainThread = UIThreadHelper.Instance.Thread;
            _sp = new TestServiceProvider();
        }
```

so basically just add DTE mock to the TestServiceProvider. Currently it has

```
        public TestServiceProvider() {
            _services.Add(typeof(SVsRegisterProjectTypes), new VsRegisterProjectGeneratorsMock());
            _services.Add(typeof(SVsRegisterEditors), VsRegisterEditorsMock.Create());
            _services.Add(typeof(IMenuCommandService), new MenuCommandServiceMock());
            _services.Add(typeof(SComponentModel), new ComponentModelMock(VsTestCompositionCatalog.Current));
            _services.Add(typeof(SVsTextManager), new TextManagerMock());
            _services.Add(typeof(SVsImageService), VsImageServiceMock.Create());
            _services.Add(typeof(SVsUIShell), new VsUiShellMock());
            _services.Add(typeof(SOleComponentManager), OleComponentManagerMock.Create());
            _services.Add(typeof(SVsSettingsManager), VsSettingsManagerMock.Create());
        }
```